experimental/dds/tree2/src/test/feature-libraries/flex-tree/lazyNode.spec.ts

```diff
@@ -599,7 +598,7 @@ function checkPropertyInvariants(root: FlexTreeEntity): void {
 	]);
 
 	const visited: Set<unknown> = new Set([root]);
-	const primitivesAndValues = new Map<PrimitiveValue | TreeValue, number>();
+	const primitivesAndValues = new Map<TreeValue, number>();
```

Comment:
Nit: rename variable to something like "treeValues"?

Reply:
Here primitives can come from places other than tree content, so those really are JS primitives (ex: type names as string end up in here, since we just walk the JS objects), so I think this name is still accurate.